### PR TITLE
Added swoole-postgres

### PR DESCRIPTION
**Pull request by doubaokun**
* Added swoole-postgres
* Updated the PHP version to be 7.4
* Updated Swoole postgres version to be 4.5.1

### CODE REVIEW

File: frameworks/PHP/swoole/benchmark_config.json [hunk 1]
```diff
@@ -24,6 +24,29 @@
       "notes": "",
       "versus": "php"
     },
+    "postgres": {
+      "json_url": "/json",
+      "plaintext_url": "/plaintext",
```
Comment (joanhey) [inline frameworks/PHP/swoole/benchmark_config.json:29]: Delete the tests not using db.

**Review by joanhey — CHANGES_REQUESTED** [review]

File: frameworks/PHP/swoole/swoole-server-postgres.php [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+    private function createDbInstance()
+    {
+        if ($this->type === 'postgres') {
+            $db = new Swoole\Coroutine\PostgreSql;
+            if ($db->connect("host={$this->server['host']} port=5432 dbname={$this->server['database']} user={$this->server['user']} password={$this->server['password']}")){
+                return $db;
+            }
+        }
+        
+        return false;
```
Comment (joanhey) [inline frameworks/PHP/swoole/swoole-server-postgres.php:235]: Why not add in `swoole-server.php` for both mysql and postgres?
So will be only one codebase to maintain.

File: frameworks/PHP/swoole/benchmark_config.json (hunk 1 above)
Comment (doubaokun) [reply frameworks/PHP/swoole/benchmark_config.json]: Removed now.

**Comment by doubaokun** [comment]
@joanhey Thanks for the review. The code is updated based on suggestion.

**Review by joanhey — APPROVED** [review]

**Pull request merged**